waltid-services/waltid-wallet-api/src/main/kotlin/id/walt/webwallet/service/SSIKit2WalletService.kt

```diff
@@ -550,16 +580,28 @@ class SSIKit2WalletService(
                 kmsType = EventDataNotAvailable
             )
         )
-        } else {
-            logger.warn { "Key delete operation not performed for alias: $alias" }
-            throw WebException(HttpStatusCode.BadRequest, "Failed to delete key: $alias")
-        }
-    }.onFailure {
-        val errorMessage = "Failed to delete key: ${it.message}"
-        logger.error(it) { errorMessage }
-        throw WebException(HttpStatusCode.BadRequest, errorMessage)
+
+        return signature
     }
 
+    override suspend fun verify(alias: String, signature: String): Boolean {
```

Comment:
"alias" looks misleading here, since it should be the actual public key jwk, maybe change it to smth. e.g. "jwk"